messages/textbox/textbox.zh-TW.yml

```diff
@@ -2,3 +2,6 @@ kendo:
   textbox:
     # The title of the clear button
     clearTitle: Clear
+
```

Comment:
@dtopalov `clearTitle` doesn't seem to be used. We can remove it as a separate fix.